build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -162,6 +162,13 @@ disableTXChecksumOffload: {{ .Values.disableTXChecksumOffload }}
 # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
 defaultMTU: {{ .Values.defaultMTU }}
 
+# packetInRate defines the OVS controller packet rate limits for different
+# features. All features will apply this rate-limit individually on sending
+# packet-in. The number stands for the rate as packets per second(pps) and the
```

Comment:
"on packet-in packets sending to antrea-agent"?

Reply:
Done.